doc/boss_tutorial.md

```diff
@@ -0,0 +1,293 @@
+# PaddlePaddle Elastic Deep Learning
+
+<img src="../logo/edl.png" width="500">
+
+While many hardware and software manufacturers are working on improving the training performance
```

Comment:
PaddlePaddle Elastic Deep Learning (EDL) is a clustering project which leverages Deep Learning training jobs to be scalable and fault-tolerant. EDL will greatly boost the parallel distributed training jobs and make good use of cluster computing power.

EDL is based on the full fault-tolerant feature of PaddlePaddle, it uses a Kubernetes controller to manage the cluster training job and an auto-scaler to scale the job's computing resources.

Reply:
for researchers ....

for enterprises ...

Reply:
Done.